### DISCUSSION TITLE
[Feature Request] Multi-domain support

### DISCUSSION THREAD

**Discussion by avenjamin** [beyondcode/herd-community#782] [Ideas]
I'd love to have a feature where you can specific additional domains, including subdomains, that a herd site can be accessed with.

In particular I want to use Laravel Subdomain routing. e.g.
```
Route::domain('admin.myapp.test')->....
```

Thanks 🙏

**Comment by michaloravec** [comment]
You can do that with `herd link`:

```
cd ~/project-directory
herd link admin.myapp
```

https://herd.laravel.com/docs/windows/1/getting-started/paths#linking-sites

**Reply by avenjamin** [reply]
Thanks for that @michaloravec. Missed that in the docs 👀

Having it in the GUI would be nice too.

**Reply by ClausMunch** [reply]
Yes, was looking for the exact same and ended up here. Would be super nice @mpociot  :)

**Reply by mechelon** [reply]
You can create a link for an existing site via the Site Wizard :) The Windows version works the same way.
![CleanShot 2025-04-23 at 08 45 00](https://github.com/user-attachments/assets/608fe879-1798-4e4c-bfec-b20609167374)

**Reply by ClausMunch** [reply]
Would be nice (and I guess easy to make) if there was just a button within the project. 
And then a list of linked domains to that project 🥰
